src/templates/plugin.routing.js

```diff
@@ -62,7 +62,9 @@ function localeRoute (route, locale) {
         !i18n.differentDomains
 
     let path = (isPrefixed ? `/${locale}${route.path}` : route.path)
-    path = path.replace(/\/+$/, '') + (trailingSlash ? '/' : '') || '/'
+    if (trailingSlash !== undefined) {
+      path = path.replace(/\/+$/, '') + (trailingSlash ? '/' : '') || '/'
+    }
```

Comment:
It's honestly quite hard for me to judge whether that change makes sense or not.

The fact that it fixes your issue seems somewhat coincidental to me since with explicit `trailingSlash=false` it would still fail.

You could say that when `trailingSlash` is undefined we should not do any trailing slash changes but then we are adding the locale prefix before that line so that already has to make a choice whether there trailing slash is added or not after locale. Currently, it adds a slash and that's what happens to be solving your issue but what it does now seems somewhat arbitrary since the idea was that the trailing slash will be "normalized" by the code you've changed.

Also with your change, `prefix` strategy and `target: 'server'` we'll redirect from `/` to `/en/` while at the same time we'll redirect from `/fr` to `/en`. So the trailing slash will be inconsistent.

So I'm not entirely convinced here... It's probably OK to make that change and it has the potential for breaking some cases also...

Reply:
If you set trailingSlash to false and set it to S3 delivery, you will get a redirect loop.
That is correct from the Nuxt settings and S3 behavior.

However, as you said, going from `/` to `/en/` is not intended.
I would like to limit it to the case of an error, as the scope of influence is large, but I don't have a good idea for that ...